sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,16 +1,12 @@
 ## Release History
 
-### 4.27.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
+### 4.27.0 (2022-03-10)
+#### Key Bugs Fixes
 * Fixed an issue in `CosmosPagedIterable` resulting in excessive memory consumption due to unbounded prefetch of pages when converting the `CosmosPagedIterable` into an `Iterator<FeedResponse<T>>`. - See [PR 27237](https://github.com/Azure/azure-sdk-for-java/pull/27237) and [PR 27299](https://github.com/Azure/azure-sdk-for-java/pull/27299)
 * Fixed a `NullPointerException` in `CosmosDiagnostics isDiagnosticsCapturedInPagedFlux` - See [PR 27261](https://github.com/Azure/azure-sdk-for-java/pull/27261)
-
-#### Other Changes
+* Fixed an issue with allowing null values for add, set and replace operations in Patch API.
```

Comment:
Q: should we be consistently add the link to PR?

Reply:
Good point, yes, we should do that, I will update it, thanks!